subprojects/groovy-jmx/src/test/java/groovy/jmx/builder/JmxConnectorHelper.java

```diff
@@ -45,7 +45,7 @@ public static Map createRmiRegistry(int initPort) {
                 port = port + 1;
                 System.out.println("JmxBuilder - *** FAILED *** to create RMI Registry - Will Retry on port [" + port + "].");
                 try {
-                    Thread.currentThread().sleep(500);
+                    Thread.sleep(500);
                 } catch (InterruptedException e) {
                     throw new RuntimeException(e);
```

Comment:
You still need to reset the interrupt flag here.

Reply:
@paulk-asert 👋 was this ever resolved?

Reply:
I created these:
https://issues.apache.org/jira/browse/GROOVY-10097
https://github.com/apache/groovy/pull/1579